src/test/java/domain/MoneyTest.java

```diff
@@ -0,0 +1,29 @@
+package domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class MoneyTest {
+    @DisplayName("같은 금액의 Money객체는 equals에 true를 반환한다.")
+    @Test
+    public void equalsTest() {
+        Money money1 = new Money("1000");
+        Money money2 = new Money("1000");
+        assertThat(money1).isEqualTo(money2);
```

Comment:
```suggestion
        Money money1 = new Money("1000");
        Money money2 = new Money("1000");

        assertThat(money1).isEqualTo(money2);
```
이렇게 한 줄씩 띄워 주면 가독성이 훨씬 좋아져요.

Reply:
[`aaf8709`](https://github.com/woowacourse/java-lotto/pull/248/commits/aaf8709de3cc71767aa4fe1a77ff62ff285ec549) 수정했습니다!